_site/en-gb/index.html

```diff
@@ -554,4 +623,16 @@ <h3 aria-label="Awarded by" class="from-label">Awarded by</h3>
     </main>
     <script src="../javascript.js"></script>
   </body>
+  <footer class="footer">
+    <p>
+      <span class="copyright-footer"
+        >©2023 Renaissance Learning, Inc. All rights reserved.</span
+      >
+      <a href="https://www.renaissance.com/privacy/">Privacy hub</a>
+      <span>|</span>
+      <a href="#" onclick="semaphore.push(['showPreferences'])"
```

Comment:
The Privacy settings link is linked to the Ketch setup, for consent and privacy management